applications/crossbar/priv/couchdb/schemas/groups.json

```diff
@@ -0,0 +1,44 @@
+{
+    "$schema": "http://json-schema.org/draft-03/schema#",
+    "_id": "groups",
+    "description": "Validator for the group",
+    "properties": {
+        "endpoints": {
+            "default": {},
+            "description": "Endpoints included into group",
+            "required": true,
+            "type": "object"
+        },
+        "music_on_hold": {
+            "default": {},
+            "description": "The music on hold parameters",
+            "name": "Music on Hold",
+            "properties": {
+                "media_id": {
+                    "description": "The ID of a media object that should be used as music on hold",
+                    "maxLength": 128,
+                    "name": "Media ID",
+                    "required": false,
+                    "type": "string"
+                }
+            },
+            "required": false,
+            "type": "object"
+        },
+        "name": {
+            "description": "A friendly name for the group",
+            "maxLength": 128,
+            "minLength": 1,
+            "name": "Name",
+            "required": true,
+            "type": "string"
+        },
+        "resources": {
```

Comment:
@lazedo, I have no idea. groups.md says it's {}, I was unable to find use of "resources" key in kazoo either.